packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -141,7 +141,7 @@ function TabsComponent(props: Props): JSX.Element {
   };
 
   return (
-    <div className="tabs-container">
+    <div className="tabs-container margin-bottom--md">
```

Comment:
Hard to say which one I like better. Seems the margins get _less consistent_?

<img width="839" alt="Screen Shot 2022-04-08 at 10 04 52 AM" src="https://user-images.githubusercontent.com/55398995/162348980-eace6f19-4f45-4815-a1b4-28f313b3ed5e.png">